cmd/sonobuoy/app/images.go

```diff
@@ -165,124 +176,180 @@ func getClusterVersion(kubeconfig Kubeconfig) (string, error) {
 	return version, nil
 }
 
-func listImages(plugin string, kubeconfig Kubeconfig) error {
-	switch plugin {
-	case "e2e":
-		version, err := getClusterVersion(kubeconfig)
-		if err != nil {
-			return errors.Wrap(err, "failed to get cluster version")
-		}
+func listImages(plugins []string, kubeconfig Kubeconfig) error {
```

Comment:
This is another good candidate for a test function; we can add it when we expand to other plugins though. I know the systemd-logs test wouldnt be very significant and the e2e one is a bit awkward to test.